assimilation_code/modules/utilities/netcdf_utilities_mod.f90

```diff
@@ -2415,7 +2415,7 @@ function nc_create_file(filename, context)
 character(len=*), parameter :: routine = 'nc_create_file'
 integer :: ret, ncid, oldmode
 
-ret = nf90_create(filename, NF90_CLOBBER, ncid)
+ret = nf90_create(filename, ior(NF90_CLOBBER,NF90_64BIT_OFFSET), ncid)
```

Comment:
Helen you probably don't want to leave  this in, see #287 for netcdf notes

Reply:
I believe we need to leave this in if we want to be able to create DART files in the netCDF-4 format (with unlimited time dim), else it will throw the error message shown in #287. 

But I'm not sure if including NF90_64BIT_OFFSET as an ior argument will cause DART files that are not in the netCDF-4 format  to be created with incorrect formatting?